plataform_front_end/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Since this is basically a setup PR, you should consider starting working on a proper README for your project, including description, how to run it, etc.